### DISCUSSION TITLE
Update to .NET 8 and AOT compatibility

### DISCUSSION THREAD

**Discussion by guillaume-chervet** [dotnet/dotNext#195] [Q&A]
Hello, thank you very much for your awesome library.
I am playing with cluster raft with asp.net core via http part and it is pretty impressive.

Did you plan to update to .NET 8 ?

I have not tried cluster part with AOT yet but i plan to use AOT to have the smaller memory foot print as possible. 

We are use Redis as database but i would like to remove it to build a really plug and play asset: https://github.com/AxaFrance/SlimFaas

We are using very few redis feature.

**Comment by sakno** [comment]
Yes, I'm waiting for .NET 8 LTS version which will come in November. I have a plan to utilize many features of C# 12 such as required properties, inline arrays etc.

AOT and IL trimming are already supported. There is no problem to use .NEXT in .NET 8 targeted solution. All assemblies are annotated accordingly for AOT compatibility.

**Reply by guillaume-chervet** [reply]
Thank you @sakno , i cannot wait for it :)

Still thank you for this awesome library!
